checker/manual/fenum-checker.tex

```diff
@@ -120,9 +118,24 @@ \section{Running the Fenum Checker\label{fenum-running}}
 
 \begin{alltt}
   javac -processor org.checkerframework.checker.fenum.FenumChecker
-        \textit{-Aquals=myproject.qual.MyFenum} MyFile.java ...
+        -Aquals=\textit{qual.MyFenum} MyFile.java ...
 \end{alltt}
 
+This option specifies a list of the fully-qualified class names of annotations,
+using a comma-no-space-separated notation.
+
+You can also provide the fully-qualified paths to a set of directories
+that contain the qualifiers using the \code{-AqualDirs} option, and add
+the directories to the boot classpath, for example:
```

Comment:
Why would I want to add the directories to the boot classpath?
Why is this not necessary for -Aquals?